kld/src/cli/client.rs

```diff
@@ -77,12 +77,12 @@ impl Api {
         &self,
         address: String,
         satoshis: String,
-        fee_rate: Option<FeeRate>,
+        fee_rate: Option<String>,
     ) -> Result<String> {
         let wallet_transfer = WalletTransfer {
             address,
             satoshis,
-            fee_rate,
+            fee_rate: fee_rate.map(|f| FeeRate::from_str(&f)).transpose()?,
```

Comment:
OpenAPI spec does not support complex types like FeeRate. So remember that it will have to be a string in the generated request type. Of course we do validation on the server side. Validation on the client is a nice to have and is difficult with clap completions.